Assets/Scripts/PlayerController.cs

```diff
@@ -5,13 +5,11 @@
 
 public class PlayerController : MonoBehaviour
 {
-    public Rigidbody rb;
     public float deathY; 
 
     private void Start()
     {
         deathY = GameController.instance.deathY;
-        rb.GetComponent<Rigidbody>();
```

Comment:
we really don't need this anymore? Does the player still move?

Reply:
yeah BallControl is the original script. This one just has the die functionality. I could have copied it to BallControl but @warrenscipio suggested we keep the script and just remove the movement stuff from here to avoid breaking stuff.